DragAndDrop/ViewController.swift

```diff
@@ -8,11 +8,16 @@
 
 import UIKit
 
-class ViewController: UIViewController {
+class ViewController: UIViewController, ViewDelegate {
 
+    var mainView:View!
```

Comment:
My other note is I wish we did not need this main view, perhaps instead of creating this the adding it as a subview we should be overriding `loadView()` to set the ViewController's `view` property to use our custom view subclass. I think this might be the correct choice.